pmm-app/tests/ia/notificationChannels_test.js

```diff
@@ -0,0 +1,67 @@
+const page = require('./pages/notificationChannelsPage');
+
+const notificationChannels = new DataTable(['name', 'type']);
+
+for (const [, channel] of Object.entries(page.types)) {
+  notificationChannels.add([channel.name, channel.type]);
+}
+
+Feature('IA: Notification Channels');
+
+
+Before(async (I, channelsAPI) => {
+  I.Authorize();
+  await channelsAPI.clearAllNCs();
```

Comment:
nit: hard to read without the context, `clearAllNotificationChannels` would read better :)

Reply:
renamed, thanks